src/js/relayclient/RelayClient.js

```diff
@@ -76,7 +76,7 @@ class RelayClient {
      */
     validateRelayResponse(returned_tx, address_relay,
                           from, to, transaction_orig, transaction_fee, gas_price, gas_limit, nonce,
-                          relay_hub_address, relay_address, sig) {
+                          relay_hub_address, relay_address, approvalData, sig) {
```

Comment:
We need to reorder these parameters so that our code conforms to some rules. In solidity, 'approvalData' goes last, why add it to the middle of parameters list here? Also in 'sendViaRelay'. 
As you are on it, please, make the parameters' order in Solidity and in JavaScript aligned. Thanks.